packages/common/src/projects/schemas.ts

```diff
@@ -0,0 +1,73 @@
+import { IConfigurationSchema, IUiSchema } from "../core";
+
+export const HubProjectSchema: IConfigurationSchema = {
+  required: ["name"],
+  type: "object",
+  properties: {
+    name: {
+      type: "string",
+      minLength: 1,
+      maxLength: 250,
+    },
+    summary: {
+      type: "string",
+      // TODO: Remove reliance on subtype as it's not valid json schema
+      // Issue https://devtopia.esri.com/dc/hub/issues/3725
+      subtype: "long-text",
+    },
+    description: {
+      type: "string",
+      // TODO: Remove reliance on subtype as it's not valid json schema
+      // Issue https://devtopia.esri.com/dc/hub/issues/3725
+      subtype: "long-text",
+    },
+  },
+} as unknown as IConfigurationSchema;
+
+/**
+ * Minimal UI Schema for Hub Project
+ */
+export const HubProjectMinimalUiSchema: IUiSchema = {
```

Comment:
This is intentionally incorrect - I just wanted something that's smaller so I can validate the function that subset's the json schema to the set of props defined in a ui schema.